src/hotspot/share/opto/partialEscape.hpp

```diff
@@ -273,6 +294,10 @@ class PEAState {
   void add_new_allocation(GraphKit* kit, Node* obj);
 
   Node* materialize(GraphKit* kit, Node* var);
+  // passive_materialize case-a
+  Node* passive_materialize(GraphKit* kit, Node* var, PEAState& newin, SafePointNode* newin_map, PhiNode* phi, RegionNode* r, uint pnum);
```

Comment:
Has no callers? Is this PR actually for case-b? Your use a/b in code, but `case1` in the PR title.

Reply:
yes. it's for Case-b.